src/Core/Prefab/GlobeView.js

```diff
@@ -163,6 +163,18 @@ export function createGlobeLayer(id, options) {
         return false;
     }
 
+    wgs84TileLayer.sumOccupancyColorLayers = (layerColors, layer) => {
```

Comment:
`wgs84TileLayer.sumOccupancyColorLayers = (colorLayers, layer) => {` is more correct.

Reply:
And an even better name would be: `countTextureSamplersForColorLayers`.
It's longer but it really describe what this function is doing.

Btw, this function (or its usage) isn't documented anywhere...